docs/articles_en/about-openvino/release-notes-openvino/release-policy.rst

```diff
@@ -1,13 +1,13 @@
 Release Policy
 =============================================================================
 
-OpenVINO offers releases of three different types, each targeting a different use case:
+OpenVINO™ offers releases of three different types, each targeting a different use case:
```

Comment:
Four (4) different types

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/25999/commits/32276d6b57d36f7ff514278c6bbd809cc7b4488f